client/components/Layout/AudioControls.vue

```diff
@@ -113,7 +113,7 @@
           </v-col>
           <v-col cols="3" class="d-none d-md-flex align-center justify-end">
             <like-button :item="getCurrentItem" class="active-button" />
-            <queue-button :item="getCurrentItem" class="active-button" />
+            <queue-button nudge-top="35" />
```

Comment:
```suggestion
            <queue-button :nudge-top="'35'" />
```
To remove the warning about passing a string to a number prop